types/office-js-preview/index.d.ts

```diff
@@ -8014,7 +8014,7 @@ declare namespace Office {
          * </table>
          *
          * The `format:` property specifies values that correspond to a subset of the settings available in the Format Cells dialog box in Excel
-         * (Right-click then select **Format Cells**, or **Home** \> **Format** \> **Format Cells**).
+         * (Select and hold (or right-click) to open the context menu then select **Format Cells**, or **Home** \> **Format** \> **Format Cells**).
```

Comment:
Consider leading with the generic outcome, then listing the mechanical actions. Also, I'd favor starting with right-click, since Windows is the overwhelming more popular add-in platform.
```suggestion
         * Open the context menu (right-click or select and hold) then select **Format Cells**, or **Home** \> **Format** \> **Format Cells**).
```